### ISSUE TITLE
Kibana workflow giving cannot find module error

### ISSUE THREAD

**Issue by thecodeassassin** [magneticio/vamp#866]
Happens with the latest katana image. Seems to be a dependency issue.

```
2016-12-19T08:59:07.306269746Z 08:59:07.297 main NOTI ==> 
2016-12-19T08:59:07.306320747Z ██╗   ██╗ █████╗ ███╗   ███╗██████╗
2016-12-19T08:59:07.306326658Z ██║   ██║██╔══██╗████╗ ████║██╔══██╗
2016-12-19T08:59:07.306329925Z ██║   ██║███████║██╔████╔██║██████╔╝
2016-12-19T08:59:07.306332831Z ╚██╗ ██╔╝██╔══██║██║╚██╔╝██║██╔═══╝
2016-12-19T08:59:07.306335866Z  ╚████╔╝ ██║  ██║██║ ╚═╝ ██║██║
2016-12-19T08:59:07.306338846Z   ╚═══╝  ╚═╝  ╚═╝╚═╝     ╚═╝╚═╝
2016-12-19T08:59:07.306341753Z                        workflow agent
2016-12-19T08:59:07.306344473Z                        version katana
2016-12-19T08:59:07.306347031Z                        by magnetic.io
2016-12-19T08:59:07.306349536Z                                        
2016-12-19T08:59:07.306352238Z 08:59:07.297 main NOTI ==> Starting Vamp Workflow Agent 
2016-12-19T08:59:07.306354925Z 08:59:07.297 main INFO ==> Key-value store type          : etcd 
2016-12-19T08:59:07.306357680Z 08:59:07.297 main INFO ==> Key-value store connection    : http://etcd-client:2379 
2016-12-19T08:59:07.306360447Z 08:59:07.297 main INFO ==> Key-value store root key path : /vamp/workflows/kibana/value 
2016-12-19T08:59:07.306363241Z 08:59:07.297 main INFO ==> Workflow file path            : /usr/local/vamp 
2016-12-19T08:59:07.306365796Z 08:59:07.297 main INFO ==> Workflow execution period     : 30 
2016-12-19T08:59:07.306369853Z 08:59:07.297 main INFO ==> Workflow execution timeout    : 10 
2016-12-19T08:59:07.306372536Z 08:59:07.297 main INFO ==> Reading workflow from         : /vamp/workflows/kibana/value 
2016-12-19T08:59:07.306375549Z 08:59:07.297 main INFO ==> Initializing etcd connection: http://etcd-client:2379 
2016-12-19T08:59:07.306385470Z 08:59:07.297 main INFO ==> etcd getting '/vamp/workflows/kibana/value' key value. 
2016-12-19T08:59:07.306388730Z 08:59:07.300 main INFO ==> Writing workflow script       : /usr/local/vamp/workflow.js 
2016-12-19T08:59:07.306391665Z 08:59:07.301 main NOTI ==> Executing workflow by Node.js. 
2016-12-19T08:59:07.579302537Z 08:59:07.579 main ERRO ==> WORKFLOW module.js:457 
2016-12-19T08:59:07.579349174Z 08:59:07.579 main ERRO ==> WORKFLOW     throw err; 
2016-12-19T08:59:07.579354215Z 08:59:07.579 main ERRO ==> WORKFLOW     ^ 
2016-12-19T08:59:07.579403109Z 08:59:07.579 main ERRO ==> WORKFLOW  
2016-12-19T08:59:07.579454323Z 08:59:07.579 main ERRO ==> WORKFLOW Error: Cannot find module 'request-promise' 
2016-12-19T08:59:07.579529714Z 08:59:07.579 main ERRO ==> WORKFLOW     at Function.Module._resolveFilename (module.js:455:15) 
2016-12-19T08:59:07.579554946Z 08:59:07.579 main ERRO ==> WORKFLOW     at Function.Module._load (module.js:403:25) 
2016-12-19T08:59:07.579596015Z 08:59:07.579 main ERRO ==> WORKFLOW     at Module.require (module.js:483:17) 
2016-12-19T08:59:07.579602889Z 08:59:07.579 main ERRO ==> WORKFLOW     at require (internal/module.js:20:19) 
2016-12-19T08:59:07.579606089Z 08:59:07.579 main ERRO ==> WORKFLOW     at Object.<anonymous> (/usr/local/vamp/workflow.js:4:12) 
2016-12-19T08:59:07.579609617Z 08:59:07.579 main ERRO ==> WORKFLOW     at Module._compile (module.js:556:32) 
2016-12-19T08:59:07.579612686Z 08:59:07.579 main ERRO ==> WORKFLOW     at Object.Module._extensions..js (module.js:565:10) 
2016-12-19T08:59:07.579615885Z 08:59:07.579 main ERRO ==> WORKFLOW     at Module.load (module.js:473:32) 
2016-12-19T08:59:07.579618923Z 08:59:07.579 main ERRO ==> WORKFLOW     at tryModuleLoad (module.js:432:12) 
2016-12-19T08:59:07.579621877Z 08:59:07.579 main ERRO ==> WORKFLOW     at Function.Module._load (module.js:424:3) 
2016-12-19T08:59:07.580287659Z 08:59:07.580 main ERRO ==> Error during execution of the workflow script. 
```

**Comment by dragoslav** [comment]
Remove local vamp-workflow-agent `katana and pull the latest one.

**Comment by thecodeassassin** [comment]
Hi Drago,

This happens a newly created cluster so i'm pretty confident i have the latest image.

**Comment by dragoslav** [comment]
It was a bug in script itself. It should be fixed in 0.9.2 release, this is valid workflow: https://github.com/magneticio/vamp-docker/blob/master/artifacts/breeds/kibana.js
